projects/unpaid-work-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/controller/personaldetails/entity/PersonalCircumstanceEntity.kt

```diff
@@ -0,0 +1,78 @@
+package uk.gov.justice.digital.hmpps.controller.personaldetails.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.Convert
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.hibernate.annotations.Immutable
+import org.hibernate.annotations.Where
+import org.hibernate.type.YesNoConverter
+import java.time.LocalDate
+
+@Entity
+@Immutable
+@Where(clause = "soft_deleted = 0 and end_date is null")
```

Comment:
Do we need to account for the end date being in the future?  Same for personal_contact.
```suggestion
@Where(clause = "soft_deleted = 0 and (end_date is null or end_date > current_date)")
```

According to pre-prod, there are >2000 personal circumstances with future end dates.